bin/offline-secrets.sh

```diff
@@ -39,6 +39,8 @@ brig:
     setNexmo: |-
       key: "dummy"
       secret: "dummy"
+    # Uncomment and update value to enable sft credentials provisioning
```

Comment:
"Provisioning" is not correct, nothing is being provisioned

Maybe something like
```suggestion
    # Uncomment to connect to SFT servers that require it```